tests/test_bijectors.py

```diff
@@ -92,3 +93,116 @@ def test_ramp_bijector_inverse_exp():
+                          {'a':0.01, 'b':0.75, 'c':0.1, 'nb_dimension':3, 'nb_component':5},
+                          {'a':1., 'b':0.05, 'c':0.99, 'nb_dimension':4, 'nb_component':2}]
+
+def test_bijectorMixtureAffineSigmoidBijector_inverse_forward():
+  """
+  Testing inverse of MixtureAffineSigmoidBijector
+  """
+  batch_size = 100
+
+  for params in _test_params_MASB:
+    x = jax.random.uniform(jax.random.PRNGKey(0), shape=[batch_size,params['nb_dimension']])
+    assert_allclose(_make_inv_forward_MixtureAffineSigmoidBijector(x, **params), x, rtol=2e-2, atol=1e-5)
+
+
+
+def _make_inv_MixtureAffineSigmoidBijector(t, a, b, c, nb_dimension, nb_component):
+  batch_size = len(t)
+
+  p = jax.random.uniform(jax.random.PRNGKey(0), shape=[batch_size,nb_dimension,nb_component])
+  p = jax.nn.softmax(p)
+
+  bijector = MixtureAffineSigmoidBijector(
+                              jnp.ones([batch_size,nb_dimension,nb_component])*a,
+                              jnp.ones([batch_size,nb_dimension,nb_component])*b,
+                              jnp.ones([batch_size,nb_dimension,nb_component])*c,
+                              p)
+
+  return bijector.inverse(t*1)
+
+
+def test_bijectorMixtureAffineSigmoidBijector_inverse_dim():
+  """
+  Testing inverse dimension of MixtureAffineSigmoidBijector
+  """
+  batch_size = 100
+
+  for params in _test_params_MASB:
+    x = jax.random.uniform(jax.random.PRNGKey(0), shape=[batch_size,params['nb_dimension']])
+    assert_allclose(_make_inv_MixtureAffineSigmoidBijector(x, **params).shape, (batch_size,params['nb_dimension']) , rtol=2e-3, atol=1e-6)
+
+
+
+def _make_forward_MixtureAffineSigmoidBijector(t, a, b, c, nb_dimension, nb_component):
+  batch_size = len(t)
+
+  p = jax.random.uniform(jax.random.PRNGKey(0), shape=[batch_size,nb_dimension,nb_component])
+  p = jax.nn.softmax(p)
+
+  bijector = MixtureAffineSigmoidBijector(
+                              jnp.ones([batch_size,nb_dimension,nb_component])*a,
+                              jnp.ones([batch_size,nb_dimension,nb_component])*b,
+                              jnp.ones([batch_size,nb_dimension,nb_component])*c,
+                              p)
+
+  return bijector.forward(t*1)
+
+_test_params_inv_MASB = [ {'a':2., 'b':0.25, 'c':0.1, 'nb_dimension':2, 'nb_component':4},
+                          {'a':0.01, 'b':0.75, 'c':0.1, 'nb_dimension':3, 'nb_component':5},
+                          {'a':1., 'b':0.05, 'c':0.99, 'nb_dimension':4, 'nb_component':2}]
+
+def test_bijectorMixtureAffineSigmoidBijector_forward_dim():
+  """
+  Testing forward dimension of MixtureAffineSigmoidBijector
+  """
+  batch_size = 100
+
+  for params in _test_params_inv_MASB:
+    x = jax.random.uniform(jax.random.PRNGKey(0), shape=[batch_size,params['nb_dimension']])
+    assert_allclose(_make_forward_MixtureAffineSigmoidBijector(x, **params).shape, (batch_size,params['nb_dimension']) , rtol=2e-3, atol=1e-6)
```

Comment:
for shapes, which are integers, it's better to use directly `assert_equal` because you don't have issues of numerical accuracy.